sale_packaging_price/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+====================
+Sale Packaging Price
+====================
+
+This module was written to extends the functionality of sale workflow
+and allow you to assign a selling price of a complete package.
```

Comment:
Include this text: "This price is split for having the price unit according the number of pieces that fits in that packaging, so there can be rounding issues to get the exact price that are warned if happens."